### DISCUSSION TITLE
Using Minotaur changelog

### DISCUSSION THREAD

**Discussion by CrypticVerse** [modrinth/.github#212] [Other discussions]
I'm using Minotaur latest version in my project, and the changelog section is a bit confusing. When I specify a file, like:

changelog = file("CHANGELOG.md")

it does not take the contents of the file, it makes it 

/home/runner/myproject/CHANGELOG.md

Does anybody know how to specify a changelog file contents without typing the changelog manually?

**Comment by triphora** [comment]
Try `project.file("CHANGELOG.md")`.

**Reply by CrypticVerse** [reply]
That gives me the error:

> Cannot set the value of extension 'modrinth' property 'changelog' of type java.lang.String using an instance of type java.io.File.

**Reply by triphora** [reply]
Sorry, you also need to add `.text` to the end. `project.file("CHANGELOG.md").text`

**Comment by CrypticVerse** [comment]
Thank You!
